pymllm/engine/launch.py

```
@@ -0,0 +1,377 @@
+            state.out_list.clear()
+            if state.finished:
+                return
+            state.event.clear()
+
+    def shutdown(self) -> None:
+        """Terminate all subprocesses."""
+        if self._rr_process is not None:
+            try:
+                self._loop.run_until_complete(self._rr_process.shutdown())
+            except Exception:
+                pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)